stripe/resource.py

```diff
@@ -936,6 +946,11 @@ def transfers(self, **params):
         return transfers
 
 
+# This resource can only be instantiated when expanded on a BalanceTransaction
+class RecipientTransfer():
```

Comment:
I suspect you're still going to have to inherit from something like `APIResource` for this to do anything. Right now it's just a perfectly vanilla class.